### ISSUE TITLE
Class subject base mark shows zero all through

### ISSUE THREAD

**Issue by toliano** [ekumahost/my-complete-high-school-portal#6]
Sir, after upload subject score for all the students in Pry 3 (for instance) and I clicked on the **_VIEW RESULT_** button (as a staff) to view the report card. Everything is fine but I noticed a default zero (0) for the **_SUBJECT BASE MARK_** column only. I tried fixing this but I couldn't

Funniest part is that, only 3 classes were having this issue but the rest of the classes were fine.

**_Oservation:_**
Whenever I input (-) for any student who is not offering a particular subject, I get this error message (incorrect INTEGER for column [score_ca1]....). And whenever I put zero (0) instead then the whole of the SUBJECT BASE column is populated with default zero. 

Please kindly help me out sir.

https://imgur.com/QdVdmXp

**Comment by ekumahost** [comment]
Looking into this now.

**Comment by kelvin019** [comment]
It appears you were putting (-) instead of just - for the CA Score. And adding a 0 to the result means that the least score will definitely be a 0. We have updated the portal now. Leave the spaces empty for students not offering the subject. If you need further assistance clearing the 0 in your database, let us know or do it yourself. Its easy.

**Comment by toliano** [comment]
Thanks so so much for your support, it is very helpful. This what happened, a student was not meant to be in pry 3 which was not removed before uploading result for the whole class, so 0 was recorded for that particular student all round in her subject since she's not meant to be in that class.
So, when I discovered she's not meant to be in that class. I went to change her status to withdraw not knowing her result from was still showing and that was why the Subject Base was populated with zeros. 

So, I had to delete her records from the (**_studentbio, std_report_card_** and **_grade_history_**) and everything came back to normal.

I can't thank you less for your support and thanks for the update too, more blessings I pray
